sympy/functions/elementary/integers.py

```diff
@@ -20,6 +20,9 @@ class RoundFunction(Function):
 
     @classmethod
     def eval(cls, arg):
+        v = cls._eval_number(arg)
+        if v is not None:
+            return v
         from sympy import im
```

Comment:
I think that the import should be at the top of the method. Also perhaps there should be a blank line after return v.

Reply:
Pay attention to things like line spacing when editing the code. Look at the code around and try to match the style

Reply:
Sorry I will not overlooked these things from now on